File: app/src/main/java/life/plank/juna/zone/data/network/model/Interaction.java
package life.plank.juna.zone.data.network.model;

import lombok.Data;

@Data
public class Interaction {
    private int likes;
    private int dislikes;
    private int shares;
    private int pins;
    private int comments;
    private int posts;
    private int blocks;
    private int bans;
    private int mutes;
    private int reports;
    private int activeUsers;
    private int followers;
}

File: app/src/main/java/life/plank/juna/zone/data/model/Interaction.kt
package life.plank.juna.zone.data.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Interaction(
        var likes: Int = 0,
        var dislikes: Int = 0,
        var shares: Int = 0,
        var pins: Int = 0,
        var comments: Int = 0,
        var posts: Int = 0,
        var blocks: Int = 0,
        var bans: Int = 0,
        var mutes: Int = 0,
        var reports: Int = 0,
        @SerializedName("emoji-reacts")
        var emojiReacts: Int = 0,
        var activeUsers: Int = 0,
        var followers: Int = 0
) : Parcelable